src/status_im/utils/contacts.cljs

```diff
@@ -17,3 +18,16 @@
                          nil)]
     (when normalized-key
       (subs (.sha3 js-dependencies/Web3.prototype normalized-key #js {:encoding "hex"}) 26))))
+
+(defn- address= [{:keys [address] :as contact} s]
```

Comment:
let's put these functions to some common namespace in order to avoid duplication

Reply:
I already moved it from src/status_im/contact/subs.cljs to src/status_im/utils/contacts.cljs so it will be still in contacts related namespaces but available to other functions, but because it is related to address I think that maybe status-im.utils.ethereum.core would be more fitting (it already has functions related to address).